extensions/amp-accordion/amp-accordion.md

```diff
@@ -89,7 +91,9 @@ to preserve the collapsed/expanded state.[/filter]
       height="256"></amp-img>
   </section>
 </amp-accordion>
+```
```

Comment:
Drive-by comment: The code opening section has four backticks, so I think you'll need 4 backticks to close it here. (Either that, or change the opening section to three backticks.)